src/components/AuthPage/ForgotPassword/index.js

```diff
@@ -45,82 +39,94 @@ const ForgotPassword = () => {
   );
 
   const onSubmit = async (values) => {
+    values.preventDefault();
     setError("");
-    await sendPasswordResetEmail(values.email)(firebase, dispatch);
+    await sendPasswordResetEmail(email)(firebase, dispatch);
   };
 
+  const theme = createMuiTheme({
+    typography: {
+        h4:{
+          fontWeight: 600,
+        }
+      },
+    palette: {
+      primary: {
+        main:"#455a64",
+      }
+    },
+  });
+
+  const useStyles = makeStyles({
+    root: {
+      padding: "2rem",
+      background: "rgba(0,0,0,.01)",
+      border: "none",
+      boxShadow: "none",
+    }
+  })
+  const classes = useStyles();
+ 
   return (
-    <Card bordered={false}>
-      <Title level={2} className="mb-24 text-center">
-        Trouble logging in?
-      </Title>
+    <ThemeProvider theme={theme}>
```

Comment:
It is good to use one `ThemeProvider` in the highest level component and make changes to it. Otherwise, this would be a nested `ThemeProvider` and the Material-UI docs say,


>   If you provide a new theme at each render, a new CSS object will be computed and injected. Both for UI consistency and performance, it's better to render a limited number of theme objects.

Read more about nesting themes 👇:
https://material-ui.com/customization/theming/#nesting-the-theme

Reply:
Removed ThemeProvider.